bindings/java/src/main/native/context.cc

```diff
@@ -47,6 +47,11 @@ Java_org_ucx_jucx_ucp_UcpContext_createContextNative(JNIEnv *env, jclass cls,
                                                          field);
     }
 
+    ucp_params.field_mask |= UCP_PARAM_FIELD_REQUEST_INIT |
+                             UCP_PARAM_FIELD_REQUEST_SIZE;
+    ucp_params.request_size = sizeof(struct jucx_context);
+    ucp_params.request_init = jucx_request_init;
```

Comment:
this was also missing in v1.6.x?

Reply:
Ah, 1.6. doesn't have this functionality at all. Not an issue then.